code/datums/helper_datums/pconstruction_datum.dm

```diff
@@ -0,0 +1,260 @@
+#define FORWARD -1
+#define BACKWARD 1
+
+/*
+This only exists so I don't break mecha construction
+because the system mecha uses sucks horribly
+
+so i'm using paradise's superior construction datum
+
+which looks 100x better in practice and is just generally better
+*/
+
+/datum/pconstruction
```

Comment:
There's really only two choices to go with this: Port spacepod construction to the horrible mess that is /tg/'s /datum/construction or port the mess that is mecha's construction (and anything else using /datum/construction) to /datum/pconstruction (and taking the p out of it)